homeassistant/components/stream/core.py

```diff
@@ -34,21 +34,20 @@ class Segment:
     sequence: int = attr.ib()
     segment: io.BytesIO = attr.ib()
     duration: float = attr.ib()
+    start_pts: tuple = attr.ib()
 
 
 class StreamOutput:
     """Represents a stream output."""
 
-    num_segments = 3
```

Comment:
Removing this breaks references in the stream record service here: https://github.com/home-assistant/core/blob/dev/homeassistant/components/stream/__init__.py#L228

Reply:
Thanks, didn't catch that.